examples/webgl_gpu_picking_point.html

```diff
@@ -0,0 +1,283 @@
+		temp *= 256;
+
+	}
+
+	// compute the depth far from the camera which between [0,far];
+	depth = camera.near * camera.far / ( camera.near * depth + camera.far * ( 1 - depth ) );
+
+	var direction = raycaster.ray.direction.normalize();
+	var target = new THREE.Vector3();
+	camera.getWorldDirection( target );
+	var dist = depth / target.dot( raycaster.ray.direction );
+
+	GPUPickingPoint.addVectors( camera.position, new THREE.Vector3().copy( direction ).multiplyScalar( dist ) );
+
+
+	if ( depth > 100 ) {
+
+		camera = pickingCamera;
+
+		var pickingCameraPosition = new THREE.Vector3();
+		var pickingCameraLookAt = new THREE.Vector3();
+		//why is 0.999 ,when depth is 10000 the point error is 1~10,if big than that value,the result could be wrong
+		pickingCameraPosition.addVectors( renderCamera.position, new THREE.Vector3().copy( direction ).multiplyScalar( ( dist - camera.near ) * 0.999 ) );
+		pickingCameraLookAt.addVectors( renderCamera.position, new THREE.Vector3().copy( direction ).multiplyScalar( dist ) );
+
+		camera.position.set( pickingCameraPosition.x, pickingCameraPosition.y, pickingCameraPosition.z );
+		camera.lookAt( pickingCameraLookAt );
+		camera.updateProjectionMatrix();
+
+
+		renderer.setRenderTarget( pickingTexture );
+		renderer.render( pickingScene, camera );
+
+		renderer.readRenderTargetPixels( pickingTexture, renderer.domElement.width / 2, renderer.domElement.height / 2, 1, 1, pixelBuffer );
+
+		//
+
+		temp = 256;
+		depth = 0;
+		for ( var i = 3; i >= 0; -- i ) {
+
+			depth += pixelBuffer[ i ] / temp;
+			temp *= 256;
+
+		}
+
+		depth = camera.near * camera.far / ( camera.near * depth + camera.far * ( 1 - depth ) );
+		GPUPickingPoint.addVectors( camera.position, new THREE.Vector3().copy( direction ).multiplyScalar( depth ) );
+
+	}
+
+
+	var intersectObject = raycaster.intersectObject( mesh );
+
+	if ( intersectObject.length ) {
+
+		console.log( "far from camera:" + dist );
+		console.log( "cpu point from camera:" + new THREE.Vector3().subVectors( camera.position, CPUPickingPoint ).length() );
+		console.log( "gpu point from camera:" + new THREE.Vector3().subVectors( camera.position, GPUPickingPoint ).length() );
+
+		console.log( "GPU picking point" );
+		console.log( GPUPickingPoint );
+
+		console.log( "CPU picking point" );
+		CPUPickingPoint.copy( intersectObject[ 0 ].point );
+		console.log( CPUPickingPoint );
+		console.log( camera.position );
+		console.log( "points distance:" + new THREE.Vector3().subVectors( CPUPickingPoint, GPUPickingPoint ).length() );
+
+		gui.updateDisplay();
+
+	}
+
+	renderer.setRenderTarget( null );
+
+}
+
+function onWindowResize() {
+
+	camera.aspect = window.innerWidth / window.innerHeight;
+	camera.updateProjectionMatrix();
+
+	renderer.setSize( window.innerWidth, window.innerHeight );
+
+}
+
+
+//
+
+function animate() {
+
+	requestAnimationFrame( animate );
+
+	render();
+	stats.update();
+
+}
+
+function render() {
+
+	controls.update();
```

Comment:
This is not needed if you remove damping. Nor is the animation loop since your scene is static.

You can enable damping again, or your can remove both the call to `update()` and the animation loop. Your choice.